rosbag_snapshot/src/snapshot.cpp

```diff
@@ -152,6 +154,7 @@ void appendParamOptions(ros::NodeHandle& nh, SnapshotterOptions& opts)
     opts.default_memory_limit_ = static_cast<int>(MB_TO_BYTES * tmp);
   if (nh.getParam("default_duration_limit", tmp))
     opts.default_duration_limit_ = ros::Duration(tmp);
+  nh.param("all", opts.all_topics_, opts.all_topics_);
```

Comment:
Would you be open to changing the parameter string to "record_all_topics"

Reply:
I'm okay, but it will be different from the commandline option (which is in turn identical to the `rosbag record` options).